.github/workflows/ci.yml

```
@@ -19,4 +19,11 @@ jobs:
       - name: Check Lint
         run: yarn lint
       - name: Build Package
-        run: yarn build
\ No newline at end of file
+        run: yarn build
+      - name: Fire Notification
+        uses: Lubycon/github-reviewer-slack-noti-action@v1.1.0
```

```suggestion
        uses: Lubycon/github-reviewer-slack-noti-action@v1.1.1
```
민석님 이거 태그가 꼬여서 1.1.0이 버그있는 버전으로 올라갔어유....흑....